doc/flatpak-metadata.xml

```diff
@@ -253,6 +268,54 @@
                                 Available since 0.3.
                             </para></listitem></varlistentry>
 
+                            <varlistentry><term><option>host-os</option></term>
+                            <listitem><para>
+                                The host operating system's libraries,
+                                executables and static data from
+                                <filename>/usr</filename>
+                                and the related directories
+                                <filename>/bin</filename>,
+                                <filename>/lib</filename>,
+                                <filename>/lib32</filename>,
+                                <filename>/lib64</filename>,
+                                <filename>/sbin</filename>.
+                                Additionally, this keyword provides access
+                                to a subset of <filename>/etc</filename> that
+                                is associated with packaged libraries and
+                                executables, even if the
+                                <option>host-etc</option> keyword
+                                was not used:
+                                <filename>/etc/ld.so.cache</filename>,
+                                (used by the dynamic linker) and
+                                <filename>/etc/alternatives</filename>
+                                (on operating systems that use it, such as
+                                Debian).
+                            </para>
+                            <para>
+                                To avoid conflicting with the Flatpak
+                                runtime, these are mounted in the sandbox
+                                at <filename>/run/host/usr</filename>,
+                                <filename>/run/host/etc/ld.so.cache</filename>
+                                and so on.
+                            </para>
+                            <para>
+                                Available since 1.7.
```

Comment:
If you want to include these in future 1.6.x branch releases I'm certainly not going to complain, although other maintainers of Flatpak in LTS distributions might.

Reply:
I branched 1.6 so this will be in 1.7